manifests/plugin/python/module.pp

```diff
@@ -0,0 +1,34 @@
+# Single module definition
+define collectd::plugin::python::module (
+  $config,
+  $script_source = undef,
+  $modulepath    = $collectd::plugin::python::modulepath,
+  $module        = $title,
+  $ensure        = 'present',
+  $order         = '50',
+  $python_conf   = "${collectd::params::plugin_conf_dir}/11-python-config.conf",
+){
+  include collectd::params
+  include collectd::plugin::python
+
+  validate_hash($config)
+
+  if $script_source {
```

Comment:
is there a case where you would want to configure a python plugin without passing a file ?